src/App.js

```diff
@@ -1,10 +1,34 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import data from './data/dummy.json';
 
 const App = () => {
+  const [response, setResponse] = useState({});
+  const [score, setScore] = useState(0);
+  const [displayScore, setDisplayScore] = useState(false);
```

Comment:
Booleans commonly have a prefix like `is`, `has` or `should` to indicate it is a boolean. We can rename this to something like `shouldDisplayScore`